src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -792,136 +794,136 @@ private AdsDataProvider() {
-                .put("src", "gpt")
-                .build()
-        },
-        {
-            new Page("project43", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/160x600"),
-            "",
-            ImmutableMap.<String, Object>builder()
-                .put("slotName", AdsContent.FLOATING_MEDREC)
-                .put("slotSize", new Dimension(160, 600))
-                .put("lineItemId", 270616092)
-                .put("src", "gpt")
-                .build()
-        },
-        {
-            new Page("project43", "SyntheticTests/Slots/Size/120x600"),
-            "",
-            ImmutableMap.<String, Object>builder()
-                .put("slotName", AdsContent.LEFT_SKYSCRAPPER_2)
-                .put("slotSize", new Dimension(120, 600))
-                .put("lineItemId", 257673852)
-                .put("src", "gpt")
-                .build()
-        },
-        {
-            new Page("project43", "SyntheticTests/Slots/Size/120x600"),
-            "",
-            ImmutableMap.<String, Object>builder()
-                .put("slotName", AdsContent.FLOATING_MEDREC)
-                .put("slotSize", new Dimension(120, 600))
-                .put("lineItemId", 257673852)
-                .put("src", "gpt")
-                .build()
-        },
-        {
-            new Page("project43", "SyntheticTests/Slots/Size/120x600"),
-            "",
-            ImmutableMap.<String, Object>builder()
-                .put("slotName", AdsContent.LEFT_SKYSCRAPPER_3)
-                .put("slotSize", new Dimension(120, 600))
-                .put("lineItemId", 257673852)
-                .put("src", "gpt")
-                .build()
-        },
-        {
-            new Page("project43", "SyntheticTests/Slots/Size/300x1050"),
-            "",
-            ImmutableMap.<String, Object>builder()
-                .put("slotName", AdsContent.MEDREC)
-                .put("slotSize", new Dimension(300, 1050))
-                .put("lineItemId", 255534972)
-                .put("src", "gpt")
-                .build()
-        },
-        {
-            new Page("project43", "SyntheticTests/Slots/Size/300x1050"),
-            "",
-            ImmutableMap.<String, Object>builder()
-                .put("slotName", AdsContent.LEFT_SKYSCRAPPER_2)
-                .put("slotSize", new Dimension(300, 1050))
-                .put("lineItemId", 255534972)
-                .put("src", "gpt")
-                .build()
-        },
+        }
+//        {
+//            new Page("project43", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/OneSkyscraper" + INSTANT_GLOBAL_ADMIX_SWITCHED_OFF),
+//            "",
+//            ImmutableMap.<String, Object>builder()
+//                .put("slotName", AdsContent.FLOATING_MEDREC)
+//                .put("slotSize", new Dimension(300, 600))
+//                .put("lineItemId", 270230292)
+//                .put("src", "gpt")
+//                .build()
+//        },
+//        {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)